lib/yle/aws/role/accounts.rb

```diff
@@ -12,7 +12,9 @@ def initialize(aliases = nil)
 
         # Returns an `AccountAlias` that best matches the passed string
         def find(id_or_alias)
-          if account_id?(id_or_alias)
+          if !id_or_alias.is_a?(String)
```

Comment:
This is a bit of anti-pattern. IMHO we could just convert the value to string what ever is passed, i.e. `id_or_alias = id_or_alias.to_s` before the `if`.